dbt_subprojects/tokens/models/prices/prices_trusted_tokens.sql

```diff
@@ -101,6 +101,7 @@ WITH trusted_tokens AS (
                 , ('ink', 0x4200000000000000000000000000000000000006)
                 , ('ink', 0xf1815bd50389c46847f0bda824ec8da914045d14)
                 , ('ink', 0x0200C29006150606B650577BBE7B6248F58470c1) -- USDT0
+                , ('ink', 0x11476323D8DFCBAFac942588E2f38823d2Dd308e) -- iETH
```

Comment:
we should have known purpose behind adding a token to trusted tokens. why add this one? is it most frequently part of pairs in dex pools?